From: Colin Walters <walters@verbum.org>
Date: Mon, 20 Jun 2016 18:13:58 -0400
Subject: libostree.sym: Fix test-symbols

The test isn't smart enough to ignore comments, so change the prefix.

Closes: #356
Approved by: jlebon
Origin: upstream, 2016.7, commit:0134c621575fcb43bb9d994251831845dac8dbd1
---
 src/libostree/libostree.sym | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libostree/libostree.sym b/src/libostree/libostree.sym
index e8cca9d..087f987 100644
--- a/src/libostree/libostree.sym
+++ b/src/libostree/libostree.sym
@@ -350,6 +350,6 @@ global:
 /*
 LIBOSTREE_2016.7 {
 global:
-        ostree_some_new_symbol;
+        someostree_some_new_symbol;
 } LIBOSTREE_2016.6;
 */
